### ISSUE TITLE
Caused by: java.lang.ClassNotFoundException: com.google.cloud.storage.Storage

### ISSUE THREAD

**Issue by alonsoir** [spring-cloud/spring-cloud-gcp#1729]
I have to reopen this issue, I am not sure if Chengyuan Zhao can see my latest message in the closed issue, so, I will repeat the same messages.

I have the same problem. I have just compiled full project doing clean install in root folder, with a build success, I tried with spring-cloud-gcp-vision-api-sample project, provisioning with a json file from gcp, and then springboot-run phase.
I am getting the same error than Abhinav1990

[stacktrace](https://pastebin.com/p7E1skC4)

[Storage class definition](https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java)

[dependency tree output](https://pastebin.com/12nkYw2G)

**Comment by meltsufin** [comment]
Duplicate of #1668 
@dzou Can you please take a look? We're probably just missing the storage dependency. Don't we have CI verifying this sample though?

**Comment by dzou** [comment]
Yes I was able to reproduce the issue. Unusual that CI did not pick this up; I ran the test manually and it failed with the same error as well.

**Comment by dzou** [comment]
Looking a bit further, I think it's due to the fact that the `storage` dependency is listed as an `optional` dependency in the vision module. Originally we listed it as optional since we needed only OCR users to provide the storage dep. However, I think we need the dependency in order for the autoconfiguration class to compile.

Turns out -dit.vision was never enabled in the travis yml. I think I forgot to enable it.

**Comment by alonsoir** [comment]
It is not merged yet into master branch, isn't?

**Comment by dzou** [comment]
Soon! :) 

We were discussing a little bit about different solution strategies. I think the current referenced PR should be satisfactory. I will wait to see if the travis build passes and move forward; should be merged in later today.

As a temporary workaround to unblock yourself, you can try adding the following to your pom.xml dependencies:
```
<dependency>
	<groupId>org.springframework.cloud</groupId>
	<artifactId>spring-cloud-gcp-storage</artifactId>
</dependency>
```

**Comment by alonsoir** [comment]
Thank you @dzou. I will wait to be merged into master branch.

**Comment by dzou** [comment]
Hi, the fix has been merged into master. Please try running it again and see if it works for you.

**Comment by alonsoir** [comment]
Hi @dzou, the ClassNotFoundException is gone, but I still have a problem. It is not related with this issue, so we can close it. 

Thank you very much.
